app/config/collections.php

```diff
@@ -4138,6 +4149,28 @@
                 'array' => true,
                 'filters' => [],
             ],
+            [
+                '$id' => ID::custom('resourceId'),
+                'type' => Database::VAR_STRING,
+                'format' => '',
+                'size' => Database::LENGTH_KEY,
+                'signed' => true,
+                'required' => false,
+                'default' => null,
+                'array' => false,
+                'filters' => [],
+            ],
+            [
+                '$id' => ID::custom('resourceType'),
+                'type' => Database::VAR_STRING,
+                'format' => '',
+                'size' => Database::LENGTH_KEY,
+                'signed' => true,
+                'required' => false,
+                'default' => null,
+                'array' => false,
+                'filters' => [],
+            ],
```

Comment:
don't we need any new indexes?

Reply:
We do not search in the migration collection by resourceId resourceType it is only a carrier of this data.